test/views/ChoiceViewSpec.scala

```diff
@@ -107,20 +107,13 @@ class ChoiceViewSpec extends ViewSpec with ChoiceMessages with CommonMessages {
 
       getElementByCss(view, "#error-message-choice-input").text() must be(messages(choiceEmpty))
     }
-
-    "display error when choice is incorrect" in {
```

Comment:
as earlier please back to original logic

Reply:
Fixed